spring-cloud-netflix-eureka-server/src/test/java/org/springframework/cloud/netflix/eureka/server/InstanceRegistryTests.java

```diff
@@ -147,14 +147,14 @@ void testRenew() {
 		assertThat(event2.getInstanceInfo()).isEqualTo(instanceInfo2);
 	}
 
-	private LeaseInfo getLeaseInfo() {
+	static LeaseInfo getLeaseInfo() {
```

Comment:
I think it would make sense moving these out to a separate class for tests so that test classes will not depend on each other (after we figured out the binder).